packages/core/src/domain/oid.ts

```diff
@@ -64,13 +66,15 @@ export const driversLicenseURIs: { [k in USState]: string } = {
   OK: `${OID_PREFIX}2.16.840.1.113883.4.3.40`,
   OR: `${OID_PREFIX}2.16.840.1.113883.4.3.41`,
   PA: `${OID_PREFIX}2.16.840.1.113883.4.3.42`,
+  PR: `n/a`,
   RI: `${OID_PREFIX}2.16.840.1.113883.4.3.44`,
   SC: `${OID_PREFIX}2.16.840.1.113883.4.3.45`,
   SD: `${OID_PREFIX}2.16.840.1.113883.4.3.46`,
   TN: `${OID_PREFIX}2.16.840.1.113883.4.3.47`,
   TX: `${OID_PREFIX}2.16.840.1.113883.4.3.48`,
   UT: `${OID_PREFIX}2.16.840.1.113883.4.3.49`,
   VA: `${OID_PREFIX}2.16.840.1.113883.4.3.51`,
+  VI: `n/a`,
```

Comment:
Couldnt find the urn for the us territories only passport urns

Reply:
I see. Is "n/a" a valid value for the places where we use the driver's license OID?